src/Tasks/Common/Resources/Strings.resx

```diff
@@ -258,6 +258,15 @@
   <data name="CannotHaveSelfContainedWithoutRuntimeIdentifier" xml:space="preserve">
     <value>It is not supported to build or publish a self-contained application without specifying a RuntimeIdentifier.  Please either specify a RuntimeIdentifier or set SelfContained to false.</value>
   </data>
+  <data name="CannotUseAppHostWithoutRuntimeIdentifier" xml:space="preserve">
+    <value>The application host cannot be used without specifying a RuntimeIdentifier. Either specify a RuntimeIdentifier or set UseAppHost to false.</value>
```

Comment:
Just a heads up: You'll need to assign error codes to these when you merge with my changes. Tests will fail to alert you to it as well.

Reply:
I'll review that PR and take the appropriate steps.  Thanks for the heads up.

Reply:
The steps should be:

1. Pick highest error code + 1 (tests will fail if you leave a gap)
2. Add `NETSDK1234`: to value like the others have now
3. Put StrBegin comment like the others too
4. Update XLF as usual.

There shouldn't be any code changes needed.